projects/laji/src/app/shared-modules/own-submissions/service/document.service.ts

```diff
@@ -51,13 +51,13 @@ export class DocumentService {
   }
 
   deleteDocument(id: string) {
-    return this.documentApi.delete(id, this.userService.getToken()).pipe(
-      mergeMap(() => this.userService.user$),
-      tap(person => {
+    const isTmpDoc = id.match(/^T:\d+$/);
```

Comment:
This could use the method `FormService.isTmpId`

Reply:
Thanks, applied